engine/src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -391,61 +399,43 @@ class SlickDataAccess(databaseConfig: Config) extends DataAccess {
     runTransaction(action)
   }
 
+  private def futureDescriptorFromExecutionAndAux(executionAndAux: ExecutionAndAux)(implicit ec: ExecutionContext): Future[WorkflowDescriptor] = {
```

Comment:
I'm always supportive of removing that 'maybe' and 'future' junk from identifier names